### DISCUSSION TITLE
[ FF7 Steam ] Battle Animations are Faster When Set to 30 or 60 FPS

### DISCUSSION THREAD

**Discussion by corigne** [julianxhokaxhiu/FFNx#571] [Q&A] [answered]
**Describe the bug**
I am using FFNx on GEProton8-3.
Edit: I have tried this on both Latest and Canary

In the .toml, I set the limiter to option 2 for 30 FPS and I noticed all battle animations are significantly sped up, regardless of back end render selected. The same is true for option 3 (60 FPS). I didn't see another bug for this posted or closed that wasn't related to 7th Heaven. I am not using 7th Heaven, just FFNx.

Edit: Is this behavior normal? I was thinking about what would happen if you just used a speed hack to increase the framerate and it's kind of in line with what I'm seeing. I hadn't really considered whether or not the frame limiter modes interpolate the frames of animation or not. My apologies if this IS intended behavior and I need another mod or something to get the animations to appear normal at 30/60 FPS. I would appreciate it if you could point me in the right direction either way.

**To Reproduce**
1. Go to FFNx.toml and enable the FPS limiter on either option 2 or 3.
2. Start the game normally.
3. Load a save or start a new game and skip the opening cutscene.
4. Enter battle
5. All battle animations are faster, but music is normal speed.

**Expected behavior**
Battle animations be at a normal speed.

**Screenshots**
I fear taking a recording while playing a game on this laptop would set it on fire.

**GPU (please complete the following information):**
- Intel i7-3667U (4) @ 3.200GHz 
- Intel Corporation 3rd Gen Core processor Graphics Controller (rev 09) (prog-if 00 [VGA controller])
	Subsystem: Apple Inc. 3rd Gen Core processor Graphics Controller
	Flags: bus master, fast devsel, latency 0, IRQ 31
	Memory at a0000000 (64-bit, non-prefetchable) [size=4M]
	Memory at 90000000 (64-bit, prefetchable) [size=256M]
	I/O ports at 2000 [size=64]
	Expansion ROM at 000c0000 [virtual] [disabled] [size=128K]
	Capabilities: <access denied>
	Kernel driver in use: i915
	Kernel modules: i915

**Additional context**
My current system info:
```bash
                   -`                    nexus@sparrow 
                  .o+`                   ------------- 
                 `ooo/                   OS: Arch Linux x86_64 
                `+oooo:                  Host: MacBookAir5,2 1.0 
               `+oooooo:                 Kernel: 6.3.3-arch1-1 
               -+oooooo+:                Uptime: 1 day, 2 hours, 52 mins 
             `/:-:++oooo+:               Packages: 1554 (pacman) 
            `/++++/+++++++:              Shell: zsh 5.9 
           `/++++++++++++++:             Resolution: 1440x900 
          `/+++ooooooooooooo/`           WM: i3 
         ./ooosssso++osssssso+`          Theme: Materia-dark-compact [GTK2/3] 
        .oossssso-````/ossssss+`         Icons: Yaru-blue-dark [GTK2/3] 
       -osssssso.      :ssssssso.        Terminal: kitty 
      :osssssss/        osssso+++.       CPU: Intel i7-3667U (4) @ 3.200GHz 
     /ossssssss/        +ssssooo/-       GPU: Intel 3rd Gen Core processor Graphics Co 
   `/ossssso+/:-        -:/+osssso+-     Memory: 4866MiB / 7852MiB 
  `+sso+:-`                 `.-/+oso:
 `++:.                           `-/+/                           
 .`                                 `/                           
```

**Comment by julianxhokaxhiu** [comment, marked as answer]
In order to make use of the 60FPS part you need to use it in tandem with 7th Heaven and the 60FPS mod there. Alone it will bring the game status to what you reported and that's expected.
